### DISCUSSION TITLE
Is it possible to send encrypted syslog to security onion?

### DISCUSSION THREAD

**Discussion by Angelozinna96** [Security-Onion-Solutions/securityonion#4328] [Unsupported Versions] [answered]
Quick question:
Security Onion is able to receive encrypted syslogs? I didn't find the answer inside the documentation :(

**Comment by facyber** [comment, marked as answer]
Have you checked Beats page? It provide some details regard sending encrypted logs over Beats for both Windows and Linux.

https://docs.securityonion.net/en/latest/beats.html

**Reply by Angelozinna96** [reply]
Oh Thank you very much!
